sycl/test-e2e/Basic/kernel_info.cpp

```diff
@@ -4,8 +4,8 @@
 // Fail is flaky for level_zero, enable when fixed.
 // UNSUPPORTED: level_zero
 //
-// Consistently fails with opencl non-fpga, enable when fixed.
-// XFAIL: opencl && !accelerator
+// Consistently fails with opencl non-fpga non-cpu, enable when fixed.
+// XFAIL: opencl && !accelerator && !cpu
```

Comment:
Instead of `XFAIL: opencl && !accelerator && !cpu` can we do `XFAIL: opencl && gpu`?

Reply:
good point, done in latest commit, thanks!